src/Jaeger/SpanConverterInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Copyright 2018 OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCensus\Trace\Exporter\Jaeger;
+
+use OpenCensus\Trace\SpanData;
+
+use Jaeger\Thrift\Span;
+
+/**
+ * This class handles converting from the OpenCensus data model into its
+ * Jaeger Thrift representation.
+ */
+interface SpanConverterInterface
```

Comment:
I don't think this interface is needed. There's only one implementation and its internal (or probably should be) until the need arises for a separate implementation.